content/docs/user-guide/how-to/undo-dvc-add.md

```diff
@@ -0,0 +1,47 @@
+# Undo dvc add
+
+There are situations where you `dvc add` a data file by mistake and want DVC to
+stop tracking that file. Follow the steps listed here to undo `dvc add`.
+
+Lets first add a data file into our example <abbr>project</abbr>:
+
+```dvc
+$ dvc add data.csv
+$ tree
+.
+├── data.csv
+└── data.csv.dvc
```

Comment:
No need for `tree` here as the directory structure isn't relevant, please use `ls`.